tests/scene/transform-tint.tests.ts

```diff
@@ -33,36 +33,36 @@ describe('Transform Tints', () =>
     {
         const root = new Container();
 
-        root._onUpdate = jest.fn();
+        (root._position as any)._onUpdate = jest.fn();
 
         root.alpha = 0.5;
 
         expect((root.alpha)).toEqual(0.5);
 
         root.alpha = 0.5;
 
-        expect(root._onUpdate).toHaveBeenCalledTimes(1);
+        expect((root._position as any)._onUpdate).toHaveBeenCalledTimes(1);
```

Comment:
Ah, this doesn't make sense, because we test `alpha` assignment here. I guess we cannot test `#`-private methods...